### PR TITLE
Data science onramp Data ingestion

### PR DESCRIPTION
**Pull request by Symmetries**
## Description

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Comment by leahecole** [comment]
CLA bot is failing on code that came from an intern who has already finished and who previously signed CLA

File: data-science-onramp/data-ingestion/setup.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    for table_name, data in EXTERNAL_TABLES.items():
+        df = spark.createDataFrame(pd.read_csv(data["url"]),
+                                   schema=data["schema"])
+
+        write_to_bigquery(df, table_name)
+
+    # Check if table exists
+    try:
+        df = spark.read.format('bigquery').option('table', TABLE).load()
+        if test:
```
Comment (leahecole) [inline data-science-onramp/data-ingestion/setup.py:148]: nit - add a comment that lets the user know what happens and why this test flag is here

File: data-science-onramp/data-ingestion/requirements-test.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+pytest==5.3.2
```
Comment (leahecole) [inline data-science-onramp/data-ingestion/requirements-test.txt:1]: should be 6.0.0 now

File: data-science-onramp/data-ingestion/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+grpcio==1.29.0
+google-auth==1.16.0
+google-auth-httplib2==0.0.3
+google-cloud==0.34.0
+google-cloud-storage==1.28.1
+google-cloud-dataproc==0.8.0
```
Comment (bradmiro) [inline data-science-onramp/data-ingestion/requirements.txt:6]: `google-cloud-dataproc==2.0.1`

**Comment by leahecole** [comment]
gonna rerun to see if it's a flake and there were other tests running

**Comment by tmatsuo** [comment]
@leahecole @bradmiro This PR deletes the top level `.gitignore`. Please add it back.

**Comment by tmatsuo** [comment]
nm, https://github.com/GoogleCloudPlatform/python-docs-samples/pull/4502

**Pull request merged**